cdap-data-fabric/src/main/java/io/cdap/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetDataset.java

```diff
@@ -274,15 +263,12 @@ private void undoPartitionDelete(DropPartitionOperation operation) throws Except
     if (srcLocation.exists()) {
       srcLocation.renameTo(files.getLocation(operation.getRelativePath()));
     }
-    // recreating the partition in Hive only makes sense if the rename succeeds
-    addPartitionToExplore(operation.getPartitionKey(), operation.getRelativePath());
   }
 
   private void undoPartitionCreate(AddPartitionOperation operation) throws Exception {
     Exception caughtExn = null;
     if (operation.isExplorePartitionCreated()) {
       try {
-        dropPartitionFromExplore(operation.getPartitionKey());
```

Comment:
Does it become an empty `try{}catch` now? Is it needed?

Reply:
Removed it